scripts/localization.ttslua

```diff
@@ -458,6 +464,597 @@ LOCALIZED_STRINGS = {
+    ["Followers"] = "Meeples",
+    ["Big Follower"] = "Grote Meeple", --"大隨從",
+    ["Builder"] = "Bouwmeester",
+    ["Pig"] = "Varken",
+    ["Mayor"] = "Burgemeester",
+    ["Wagon"] = "Transportwagen",
+    ["Barn"] = "Herenboerderij",
+    ["Shepherd"] = "Herder",
+    ["Messenger"] = "Boodschapper",
+    ["Mage"] = "Tovenaar",
+    ["Mages"] = "Tovenaars", --TEMP: for model substitution
+    ["Witch"] = "Heks",
+    ["Witches"] = "Heksen", --TEMP: for model substitution
+    ["Robber"] = "Rover", -- From mini-expansion: The Robbers
+    ["Robbers"] = "Rovers", -- From mini-expansion: The Robbers
+    --Situational Figure Names
+    ["Meeple"] = "Meeple", --one of the 7 basic followers
+    ["Farmer"] = "Boer", --follower in a field
+    ["Knight"] = "Ridder", --follower in a city
+    --["Robber"] = "Struikrover", --follower on a road - Shared with Robber name from The Robber
+    ["Monk"] = "Monnik", --follower in a cloister
+    --Neutral Figure Names
+    ["Dragon"] = "Draak",
+    ["Fairy"] = "Fee",
+    ["Count"] = "Graaf",
+    ["Tower"] = "Toren",
+    ["Towers"] = "Torens",
+    ["Bridge"] = "Brug",
+    ["Bridges"] = "Bruggen",
+    ["Castle"] = "Kasteel",
+    ["Castles"] = "Kastelen",
+    ["Gold Bar"] = "Goudstaaf",
+    --Feature Names
+    ["Field"] = "Weide",
+    ["Fields"] = "Weides",
+    ["City"] = "Stad",
+    ["Cities"] = "Steden",
+    ["Road"] = "Weg",
+    ["Roads"] = "Wegen",
+    ["Cloister"] = "Klooster",
+    ["Cloisters"] = "Kloosters",
+    --Situational Feature Names
+    ["Farm"] = "Boerderij", --field with a follower on it
+    ["Farms"] = "Boerderijen", --field with a follower on it
+    --Special Feature Names
+    ["Cathedral"] = "Kathedraal",
+    ["Cathedrals"] = "Kathedralen",
+    ["Inn"] = "Herberg",
+    ["Inns"] = "Herbergen",
+    ["Princess"] = "Jonkvrouw",
+    ["Volcano"] = "Vulkaan",
+    ["Portal"] = "Poort", --from big box 3 manual (bad translation?)
+    ["Portals"] = "Poorten",
+    ["Tower Foundation"] = "Torenbouwplaats",
+    ["Abbey"] = "Abdij",
+    ["Shrine"] = "Cultusplaats",
+    ["Shrines"] = "Cultusplaatsen",
+    ["Hill"] = "Heuvel",
+    ["Hills"] = "Heuvels",
+    ["Vineyard"] = "Wijngaard",
+    ["Vineyards"] = "Wijngaarden",
+    ["Bazaar"] = "Bazaar",
+    ["Bazaars"] = "Bazaars",
+    ["Fliers"] = "Vliegtuigen",
+    ["Messages"] = "Nieuwsberichten",
+    ["Ferries"] = "Veerboten",
+    ["Gold"] = "Goud", --Should the special feature just be called gold bars?
+    --City of Carcassonne Quarters
+    ["Castle Quarter"] = "Kasteel",
+    ["Blacksmith Quarter"] = "Smederij",
+    ["Cathedral Quarter"] = "Kathedraal",
+    ["Market Quarter"] = "Markt",
+    --Wheel of Fortune Features
+    ["Crown"] = "Kroon",
+    ["Fortune"] = "Fortuin",
+    ["Taxes"] = "Belasting",
+    ["Famine"] = "Hongersnood",
+    ["Storm"] = "Storm",
+    ["Inquisition"] = "Inquisitie",
+    ["Plague"] = "Pest",
+    --["Dragon"] = "火龍", --shared with figure name

```

Comment:
I think this can be removed. I had left it in case the figure "Dragon" would somehow be translated differently than the "Dragon" icon on the tiles.